test/support/schemas/prefix_schema.exs

```diff
@@ -3,12 +3,12 @@ defmodule PrefixSchema do
   use Absinthe.Schema
 
   query do
-    field :foo do
+    field :foo, :integer do
```

Comment:
@bruce we may want to have some additional checks in the field macro that make sure a type of _some_ kind is set. Honestly I'd just raise an exception, it's basically a function call error.